firebase.json

```diff
@@ -0,0 +1,16 @@
+{
+  "hosting": {
+    "public": "build",
+    "ignore": [
```

Comment:
Is this enough for firebase tools to automatically run `npm i` and `npm run build`?

Reply:
Fixed, also apparently the `.github` directory was missing :)